python/cog/predictor.py

```diff
@@ -283,12 +308,20 @@ def validate_input_type(
             f"No input type provided for parameter `{name}`. Supported input types are: {readable_types_list(ALLOWED_INPUT_TYPES)}, or a Union or List of those types."
         )
     if type not in ALLOWED_INPUT_TYPES:
-        if get_origin(type) in (Union, List, list) or (
+        if get_origin(type) is Literal:
```

Comment:
we should check if the AST parser can deal with Literals

Reply:
Added by 095fd997cfdeb2ccf94bd63213b254dfad3cee9d